### ISSUE TITLE
2023 Design 02

### ISSUE THREAD

**Issue by lmas3009** [lmas3009/design2code2live#125]
Create a folder with the Design 01/{yourname} 
Combined code with part 1

Optional: Make it Responsive

Note: Add your images, about, live GitHub/dev.to links. **PART 1**
Design 02 is **PART 2**

We accept multiple PR's. This is your personal website. 

**✨✨Do not add the same text as shown**

Design by **Aravind Kumar Vemula**

## Design
![MacBook Pro 16_ - 4](https://github.com/lmas3009/design2code2live/assets/60876387/d3cc6dd1-ea97-4cea-9816-918ffb13360f)

## Resources

fonts: https://fonts.google.com
colors: https://coolors.co
images: https://unsplash.com or https://www.pexels.com

**Comment by Aka-suvam** [comment]
Hey   Imas,  I'm  new  to  open  source  and  currently  learning  web  development,  i  have  also  done  small  project using- HTML/CSS .
But i dont know Tailwind-Css  .  I can built this page using HTML and CSS and can also make it responsive .  can you assign me this issue.

Thanks.

**Comment by lmas3009** [comment]
Yes, you can use CSS. Tailwind CSS is optional. But this issue can have multiple assigned people. This issue is to create 
 your own personal branding page.

**Comment by Aka-suvam** [comment]
hey imas sorry for being late but i have made  the mention changes . you take a look at the screen shot . then i will make a Pull req.

![Screenshot](https://github.com/lmas3009/design2code2live/assets/94453074/7e09204d-a16b-4dd9-b761-40c65c216e70)

**Comment by lmas3009** [comment]
@Aka-suvam  remove the logo at the top and raise a PR and the folder format will be Design 01_2023/{yourname}

**Comment by siddhant2002** [comment]
Can I work under this issue?

**Comment by Skb08** [comment]
@lmas3009 I also want to contribute. Can you assign me?

**Comment by lmas3009** [comment]
sure @Skb08

**Comment by siddhant2002** [comment]
Can I work under this issue?

**Comment by siddhant2002** [comment]
Can I work under this issue?

**Comment by siddhant2002** [comment]
Can I work under this issue @lmas3009 ?

**Comment by lmas3009** [comment]
@siddhant2002

**Comment by Prithvi-R** [comment]
I can built this page using HTML and CSS. Can I work under this issue?? @lmas3009

**Comment by abhi-yo** [comment]
hey, @lmas3009 can you please assign this to me
